net-misc/dahdi-tools/dahdi-tools-3.1.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools bash-completion-r1
+
+DESCRIPTION="Userspace tools to configure the kernel modules from net-misc/dahdi"
+HOMEPAGE="https://www.asterisk.org"
+SRC_URI="https://downloads.asterisk.org/pub/telephony/${PN}/releases/${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE="ppp"
+PATCHES=( "${FILESDIR}/dahdi-nondigium-blacklist.patch" )
+
+DEPEND="dev-libs/newt
+	net-misc/dahdi
+	sys-kernel/linux-headers
+	virtual/libusb:0
+	ppp? ( net-dialup/ppp )"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf $(use_with ppp)
+}
+
+src_install() {
+	local bashcompdir="$(get_bashcompdir)"
+	local bashcmd bashcmdtarget
+
+	emake DESTDIR="${D}" bashcompdir="${bashcompdir}" udevrulesdir=/lib/udev/rules.d install
+	emake DESTDIR="${D}" install-config
```

Comment:
This is throwing ACCESS_DENIED errors on me, 
```
###################################################
###
### DAHDI tools installed successfully.
### If you have not done so before, install init scripts with:
###
###   make install-config
###
###################################################

make -j1 DESTDIR=/tmp/portage/net-misc/dahdi-tools-3.1.0/image install-config 
mkdir -p /tmp/portage/net-misc/dahdi-tools-3.1.0/image/etc/dahdi
mkdir -p /tmp/portage/net-misc/dahdi-tools-3.1.0/image/etc/modprobe.d
install -m644 xpp/genconf_parameters /tmp/portage/net-misc/dahdi-tools-3.1.0/image/etc/dahdi/genconf_parameters
install -m644 init.conf.sample /tmp/portage/net-misc/dahdi-tools-3.1.0/image/etc/dahdi/init.conf
install -m644 blacklist.sample /tmp/portage/net-misc/dahdi-tools-3.1.0/image/etc/modprobe.d/dahdi-blacklist.conf
install -m644 modprobe.conf.sample /tmp/portage/net-misc/dahdi-tools-3.1.0/image/etc/modprobe.d/dahdi.conf
make -f ./Makefile.legacy top_srcdir=. srcdir=. config
make[1]: Entering directory '/tmp/portage/net-misc/dahdi-tools-3.1.0/work/dahdi-tools-3.1.0'
DAHDI has been configured.

List of detected DAHDI devices:

 * ACCESS DENIED:  fopen_wr:     /dev/stderr
tee: /dev/stderr: Permission denied
No hardware found
make[1]: Leaving directory '/tmp/portage/net-misc/dahdi-tools-3.1.0/work/dahdi-tools-3.1.0'
>>> Completed installing net-misc/dahdi-tools-3.1.0 into /tmp/portage/net-misc/dahdi-tools-3.1.0/image

 * Final size of build directory: 14636 KiB (14.2 MiB)
 * Final size of installed tree:   2380 KiB ( 2.3 MiB)

 * --------------------------- ACCESS VIOLATION SUMMARY ---------------------------
 * LOG FILE: "/var/log/sandbox/sandbox-4.log"
 * 
VERSION 1.0
FORMAT: F - Function called
FORMAT: S - Access Status
FORMAT: P - Path as passed to function
FORMAT: A - Absolute Path (not canonical)
FORMAT: R - Canonical Path
FORMAT: C - Command Line

F: fopen_wr
S: deny
P: /dev/stderr
A: /dev/stderr
R: /dev/stderr
C: tee /dev/stderr 
 * --------------------------------------------------------------------------------

>>> Failed to emerge net-misc/dahdi-tools-3.1.0
```

And before that I got
```
 * QA Notice: make jobserver unavailable:
 * 
 * 	make[3]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
 * 	make[2]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
 * 	make[2]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
 * 	make[4]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
 * 	make[1]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
```
hence the -j1. 

What happens for you if you closely observe src_install phase in your build.log?

Reply:
Hmm.  That's during config-install.  Anyway ... , LEGACY_MAKE = $(MAKE) ... *sigh* ... doubt that + that's required will be accepted upstream, and definitely not your tee resulting in the /dev/stderr access violation (although, for that I just stopped make install-config from doing $(MAKE_LEGACY) config - which does stuff we don't need nor want).

I can't reproduce, do you mind?

Then, I believe this has always been the case, but it's likely wrong.  dahdi-tools installs a handful of perl modules into /usr/local/lib64/perl5 ... this has been updated to /usr/lib64 now with the help of the perl-functions eclass.  Please confirm this is in order.